go/pkg/outerr/outerr_test.go

```diff
@@ -0,0 +1,62 @@
+package outerr
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"testing"
+)
+
+func TestRecordingOutErr(t *testing.T) {
+	t.Parallel()
+	o := NewRecordingOutErr()
+	o.WriteOut([]byte("hello"))
+	o.WriteErr([]byte("world"))
+	gotOut := o.GetStdout()
+	if !bytes.Equal(gotOut, []byte("hello")) {
+		t.Errorf("expected o.GetStdout() to return hello, got %v", gotOut)
+	}
+	gotErr := o.GetStderr()
+	if !bytes.Equal(gotErr, []byte("world")) {
+		t.Errorf("expected o.GetStderr() to return hello, got %v", gotErr)
+	}
+}
+
+func TestSystemOutErr(t *testing.T) {
+	t.Parallel()
```

Comment:
Is it safe to run this test in parallel though since its redirecting stdout and stderr of this process?

Reply:
Good point! It is now, because there's only one test case that does it. But if we ever add another test case here, that may no longer be true.

Reply:
I'd suggest removing it since its likely we'll forget this exists when adding a new test case and wonder why test output is flaky :)

Reply:
Sure, done.